lib/classes/manage_location.rb

```diff
@@ -0,0 +1,20 @@
+class ManageLocation
+  def initialize(location_id)
+    @location = Location.find_by(id: location_id)
+  end
+
+  def empty_location?
+    @location.packages_locations.count.zero? &&
+      @location.package_types.count.zero? &&
+      PackagesInventory.where(location_id: @location.id).count.zero? &&
```

Comment:
There will likely be cases where the we have packages_inventories records, but the quantity is 0. Those wouldn't be detected with this, right ?

Reply:
yes @patrixr , those records will not be detected.